zappa/zappa.py

```diff
@@ -752,8 +749,8 @@ def create_and_setup_methods(self, restapi, resource, lambda_arn, api_key_requir
             integration.Credentials = credentials
             integration.IntegrationHttpMethod = 'POST'
             integration.IntegrationResponses = []
-            integration.PassthroughBehavior = 'NEVER'
-            integration.RequestParameters = {}
+            # integration.PassthroughBehavior = 'NEVER'
```

Comment:
Can't say right now; some side effect of my experiments.
I was able to force ApiGateway to forward real url to Zappa in some conditions, but didn't have the time to finish it. I think it may be returned back.